### PR TITLE
.ci/aws: Pin p4/p5 ami's to AMI's from 8/7/24

### PR DESCRIPTION
**Pull request by a-szegel**
Attempt to stabilize the CI

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rauteric — DISMISSED** [review]

File: .ci/aws/Jenkinsfile [hunk 1]
```diff
@@ -219,21 +219,25 @@ pipeline {
...
                     stages["4_p3dn_ubuntu2204"] = get_test_stage_with_lock("4_p3dn_ubuntu2204", env.BUILD_TAG, "ubuntu2204", "p3dn.24xlarge", p3dn_region, p3dn_lock_label, num_instances, p3dn_odcr, p3dn_addl_args)
 
                     // p4d tests
-                    stages["4_p4d_alinux2"] = get_test_stage_with_lock("4_p4d_alinux2", env.BUILD_TAG, "alinux2", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, p4d_odcr, p4_p5_addl_args)
-                    stages["4_p4d_ubuntu2004"] = get_test_stage_with_lock("4_p4d_ubuntu2004", env.BUILD_TAG, "ubuntu2004", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, p4d_odcr, p4_p5_addl_args)
-                    stages["4_p4d_ubuntu2204"] = get_test_stage_with_lock("4_p4d_ubuntu2204", env.BUILD_TAG, "ubuntu2204", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, p4d_odcr, p4_p5_addl_args)
+                    stages["4_p4d_alinux2"] = get_test_stage_with_lock("4_p4d_alinux2", env.BUILD_TAG, "alinux2", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, p4d_odcr, p4_p5_addl_args + al2_ami)
```
Comment (sunkuamzn) [inline .ci/aws/Jenkinsfile:233]: AMIs are in the same region as p5 capacity but not p4d

**Review by sunkuamzn — CHANGES_REQUESTED** [review]
p5 and p4d capacity are not in the same region, so need different AMIs

**Review by sunkuamzn — APPROVED** [review]

**Comment by a-szegel** [comment]
bot:aws:retest... looks like we stumbled onto a hardware problem

**Comment by a-szegel** [comment]
closing this.... we still see instability in this... we need to fix forward, not roll backwards.

**Pull request closed without merging**